utils/node_local_test.py

```diff
@@ -1199,14 +1223,18 @@ def run_daos_cmd(conf,
     exec_cmd.extend(cmd)
 
     cmd_env = get_base_env()
+    if not log_check:
+        del cmd_env['DD_MASK']
+        del cmd_env['DD_SUBSYS']
+        del cmd_env['D_LOG_MASK']
+
+    with tempfile.NamedTemporaryFile(prefix='dnt_cmd_{}_'.format(get_inc_id()),
```

Comment:
Should this be using `with`? As described in https://docs.python.org/3/library/tempfile.html#examples the temporary file created will be removed once we leave the `with` context and `cmd_env['D_LOG_FILE']` will point to non-existent file.

Reply:
This code passes the delete=False flag to NamedTemporaryFile so it will not get deleted when the context is destroyed, we only need lf for long enough to read lf.name

Reply:
Ah sorry missed the `delete=False` flag.